benchmarking/evaluations/graphing/line_graph.py

```diff
@@ -56,6 +73,13 @@ def line_graph(graph_data: LineGraphMetadata):
     if graph_data.y_label is not None:
         plt.ylabel(graph_data.y_label)
 
+    if graph_data.num_minor_ticks:
+        ax[0][0].xaxis.set_minor_locator(
+            AutoMinorLocator(graph_data.num_minor_ticks + 1)
+        )
+    else:
+        ax[0][0].xaxis.set_minor_locator(AutoMinorLocator())
```

Comment:
joke; you have a way to auto locate minors? 🤨

Reply:
Haha 😂